libraries/AdaptiveExpressions/Expression.cs

```diff
@@ -762,49 +789,102 @@ public class FunctionTable : IDictionary<string, ExpressionEvaluator>
+            /// </summary>
+            /// <param name="key">A string key.</param>
+            /// <returns>A boolean value.</returns>
+            public bool ContainsKey(string key) => ExpressionFunctions.StandardFunctions.ContainsKey(key) || this._customFunctions.ContainsKey(key);
+
+            /// <summary>
+            /// Not implemented.
+            /// </summary>
+            /// <param name="array">An array of string values.</param>
+            /// <param name="arrayIndex">An integer of index.</param>
             public void CopyTo(KeyValuePair<string, ExpressionEvaluator>[] array, int arrayIndex) => throw new NotImplementedException();
 
-            public IEnumerator<KeyValuePair<string, ExpressionEvaluator>> GetEnumerator() => ExpressionFunctions.StandardFunctions.Concat(this.customFunctions).GetEnumerator();
-
-            public bool Remove(string key) => this.customFunctions.TryRemove(key, out var oldVal);
-
+            /// <summary>
+            /// Generate an enumator through all standard functions.
+            /// </summary>
+            /// <returns>An enmurator of standard functions.</returns>
+            public IEnumerator<KeyValuePair<string, ExpressionEvaluator>> GetEnumerator() => ExpressionFunctions.StandardFunctions.Concat(this._customFunctions).GetEnumerator();
+
+            /// <summary>
+            /// Remove a specified key from user customized functions.
+            /// </summary>
+            /// <param name="key">A string key of function name.</param>
+            /// <returns>A boolean value.</returns>
+            public bool Remove(string key) => this._customFunctions.TryRemove(key, out var oldVal);
+
+            /// <summary>
+            /// Remove a specified key value pair from user customized functions.
+            /// </summary>
+            /// <param name="item">A key value pair of string to ExpressionEvaluator.</param>
+            /// <returns>A boolean value.</returns>
             public bool Remove(KeyValuePair<string, ExpressionEvaluator> item) => Remove(item.Key);
 
+            /// <summary>
+            /// Attempts to get the value associated with the specified key from the FunctionTable.
+            /// </summary>
+            /// <param name="key">The key of the value to get.</param>
+            /// <param name="value">When this method returns, contains the object from the FunctionTable
+            /// that has the specified key, or the default value of the type if the operation failed.</param>
+            /// <returns> A boolean value. </returns>
             public bool TryGetValue(string key, out ExpressionEvaluator value)
             {
                 if (ExpressionFunctions.StandardFunctions.TryGetValue(key, out value))
                 {
                     return true;
                 }
 
-                if (this.customFunctions.TryGetValue(key, out value))
+                if (this._customFunctions.TryGetValue(key, out value))
```

Comment:
Since we are changing private fields to underscores, we can also remove "this.". Please consider removing them for all these changes.